action.yml

```diff
@@ -13,6 +13,10 @@ inputs:
     description: number of times to attempt fetching the commit status
     required: false
     default: 10
+  enable_a11y_threshold:
+    description: Enable the a11y threshold, which will cause the action to fail if the number of violations is greater than the threshold
+    required: false
+    default: "false"
```

Comment:
IMO this should either be of `type: choice` or `type: boolean`. See https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_dispatchinputs

Reply:
Oh I should have checked the docs, because I tried the boolean initially instead of a string and the GHA yml hates it: 

<img width="789" alt="image" src="https://github.com/dequelabs/axe-devhub-action/assets/41127686/9fcb8910-49d7-4d9e-bf53-8289136365d5">

I'll update it regardless of the lint error, as I agree it should be a `boolean` and not a `string`

Reply:
Looks like we can't use `type: ` here, only for manually dispatched workflows 😬 

Sorry!